docs/docsource/installation/configure/configure_cosmos_for_OOB.rst

```diff
@@ -143,136 +143,113 @@ parameters can be provided while installing the chart. For example,
       Use the default values.yaml file from the Zenko
       source at ~/Zenko/kubernetes/cosmos/values.yaml
 
-Configure Cosmos on a Zenko Instance
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- 
-To configure Cosmos on a Zenko instance, perform the following steps
-from `Orbit <https://admin.zenko.io>`_.
- 
+Configure Cosmos
+----------------
+
+To configure Cosmos with a Zenko instance:
+
```

Comment:
Honestly, I think having the Cloudserver configuration here confuses and clutters these documents. Otherwise the configuration is simple with a few steps with limited subset of options (all done from Orbit). That being said, Cosmos has it's own simple `README` where these things are described which is sufficient for any open source users that may be using a standalone Cloudserver and Cosmos configuration.